src/main/java/seedu/duke/chatbot/Storage.java

```diff
@@ -0,0 +1,192 @@
+package seedu.duke.chatbot;
+
+import seedu.duke.exceptions.DukeException;
```

Comment:
I like that you have already organised the different classes into separate packages, I think this will definitely help with the organisation of files as the project scales up! 😄